src/rust/engine/src/fs.rs

```diff
@@ -16,6 +16,8 @@ use tar;
 use tempdir::TempDir;
 
 use hash::{Fingerprint, WriterHasher};
+use util;
```

Comment:
`util` catchalls are generally not a good idea... in particular, the utility functions added over there would probably make more sense as static functions in `fs` instead. Maybe just move them out of `Snapshots` and to the top level?